torch/_decomp/__init__.py

```diff
@@ -283,16 +272,8 @@ def core_aten_decompositions() -> Dict[OpOverload, Callable]:
             aten.nan_to_num,
             aten.nan_to_num_,
             aten.narrow,
-            aten.native_batch_norm,
```

Comment:
The story around batch_norm is a bit complicated... 
Ideally, all variants of batch_norms should be decomp into "the_one_batch_norm"... 
But I am not sure which variant is "the_one_batch_norm"? 

I currently marked "aten.native_batch_norm" as core, but this is not necessarily correct.

Reply:
Ok, I can add native_batch_norm as part of our op discussion list.